spec/models/accounting/interest_calculator_spec.rb

```diff
@@ -3,7 +3,7 @@
 # See docs/example_calculation.xlsx for ground truth used to build this spec.
 describe Accounting::InterestCalculator do
   let!(:division) { create(:division, :with_accounts) }
-  let(:loan) { create(:loan, division: division, rate: 8.0) }
+  let(:loan) { create(:loan, division: division, rate: 8.0, status_value: 'active') }
```

Comment:
Is there a way to handle this and any other related properties that might need to be set as a trait on the factory? It seems like this is something we should be using in other tests too.